examples/complete/main.tf

```diff
@@ -37,3 +37,71 @@ module "create_scc_instance" {
   en_instance_crn                   = module.event_notification.crn
   skip_cos_iam_authorization_policy = false
 }
+
+module "create_scc_controls" {
+  source                      = "../../controls/."
+  instance_id                 = module.create_scc_instance.guid
+  control_library_name        = "control_library_name"
+  control_library_description = "control_library_description"
+  control_library_type        = "custom"
+  latest                      = true
+  version_group_label         = "de38e8c4-2212-4e4b-8dcf-b021b98d8e43"
+  controls = [
+    {
+      control_id          = "032a81ca-6ef7-4ac2-81ac-20ee4a780e3b"
```

Comment:
Why would we be passing a control ID? Are we not creating a new control here?

Reply:
just to test the example on reapply, need to wait for data lookup so we can get existing control ids